TS04-Dplyr-And-Time-Series-In-R.Rmd

```diff
@@ -356,14 +475,25 @@ harTemp.daily.09.11<-harMet15.09.11 %>%
 head(harTemp.daily.09.11)
 ```
 
-###Carry Date-Time Data with Mutated Data
-Now that we have the `harTemp.daily.09.11` data frame we may want to plot the
-air temperature throughtout the year.  But what would be our x-axis?  
+###Add Date-Time To DPLYR Output
```

Comment:
Fixed. We might create an issue as it's useful to capitalize in titles potentially. But consistency is important.